app/experimenter/nimbus-ui/src/components/PageEditMetrics/index.tsx

```diff
@@ -5,11 +5,37 @@
 import React from "react";
 import { RouteComponentProps } from "@reach/router";
 import AppLayoutWithExperiment from "../AppLayoutWithExperiment";
+import FormMetrics from "../FormMetrics";
+import LinkExternal from "../LinkExternal";
+import { useConfig } from "../../hooks/useConfig";
 
 const PageEditMetrics: React.FunctionComponent<RouteComponentProps> = () => {
+  const { probeSets } = useConfig();
+
   return (
     <AppLayoutWithExperiment title="Metrics" testId="PageEditMetrics">
-      {({ experiment }) => <p>{experiment.name}</p>}
+      {({ experiment }) => (
+        <>
+          <p>
+            Every experiment analysis automatically includes how your experiment
+            has impacted <strong>Retention, Search Count, and Ad Click</strong>{" "}
+            metrics. Get more information on{" "}
+            <LinkExternal href="">Core Firefox Metrics.</LinkExternal>
```

Comment:
I'm surprised eslint-plugin-jsx-a11y didn't bark at you for not supplying an href. Maybe we can throw a TODO note above referencing EXP-656